taverna-scufl2-api/src/test/java/org/apache/taverna/scufl2/api/common/TestScufl2Tools.java

```diff
@@ -51,6 +51,7 @@
 import org.apache.taverna.scufl2.api.profiles.Profile;
 import org.junit.Before;
 import org.junit.Test;
+import sun.security.krb5.Config;
```

Comment:
I don't think we need `sun.security.krb5.Config` (or that it's even legal to import)